src/MainPage/MainPage.js

```diff
@@ -148,106 +154,81 @@ function MainPage() {
       },
       body: JSON.stringify({ energy: newEnergy })
     })
-      .then(response => response.json())
-      .then(data => {
-        console.log("Energy updated:", data);
-        setPokemonData(data)
-      })
-      .catch(error => console.error("Error updating energy:", error));
+    .then(response => response.json())
+    .then(data => {
+      console.log("Energy updated:", data);
+      setPokemonData(data)
+    })
+    .catch(error => console.error("Error updating energy:", error));
   };
 
-
-  useEffect(() => {
-    fetchData(2);
-  }, [])
-
-  // function triggerHops(){
-  //   setPlayAnim(1);
-  //   console.log(playAnim);
-  // }
   function playWithCurrentPokemon() {
+  
+    let pokemonSprite = document.getElementById("currentRender")
 
-    console.log(playAnim)
-    setPlayAnim(1);
-
-    console.log(playAnim)
+    pokemonSprite.classList.remove('jump')
```

Comment:
They said it couldnt be done! Very smart work around for our animation problems using a toggle and timeout, I will use this in future projects.